apps/web/src/components/ExperienceFormFields/EducationFields.tsx

```diff
@@ -140,14 +124,7 @@ const EducationFields = ({ labels }: SubExperienceFormProps) => {
             label={labels.thesisTitle}
             name="thesisTitle"
             type="text"
-            rules={{
-              maxLength: {
-                message: intl.formatMessage(errorMessages.overCharacterLimit, {
-                  value: 256,
-                }),
-                value: 255,
-              },
-            }}
+            rules={{ required: intl.formatMessage(errorMessages.required) }}
```

Comment:
Thesis title isn't required

Reply:
Oops 😅 Reverted in [revert making thesis title required](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8894/commits/d1cc4b3c7de84d9b22e2175b3a9b816274869185)